### PR TITLE
feat: improve deploy scripts

### PR DESCRIPTION
**Pull request by kartojal**
- Add Goerli support
- Improve initialization script to skip already initialized tokens in case of a RPC network error.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +37,7 @@
     "amm:kovan:full:migration": "npm run compile && npm run hardhat:kovan -- amm:mainnet --skip-registry",
     "aave:docker:full:migration:add-registry": "npm run compile && npm run hardhat:docker -- aave:mainnet",
     "aave:kovan:full:migration:add-registry": "npm run compile && npm run hardhat:kovan -- aave:mainnet",
+    "aave:goerli:full:migration:add-registry": "npm run compile && npm run hardhat:goerli -- aave:testnet",
```
Comment (miguelmtzinf) [inline package.json:40]: Do we wanna use `aave:testnet` instead of `aave:mainnet`?

**Review by miguelmtzinf — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kartojal) [reply package.json:40]: Yes, due at the moment of the deployment of Aave V2 at the Goerli testnet there was no Chainlink oracles, a main dependency to run the aave:mainnet scripts.

The aave:testnet scripts creates a oracle mock system to impersonate Chainlink oracles.

Comment (miguelmtzinf) [reply package.json:40]: Makes sense, thanks!

**Comment by BEIFENGLE4** [comment]
So nice!

**Pull request merged**